portfolio/src/main/java/com/google/sps/servlets/MarkerServlet.java

```diff
@@ -99,4 +125,79 @@ public void storeMarker(Marker marker) {
 
     datastore.put(markerEntity);
   }
+
+  public boolean repeatMarkers(double reportLat, double reportLng, String reportDate,
+   String reportTime, String reportCrime) {
```

Comment:
Not a great name, maybe should be called validateMarker, and it should return true if the marker is valid.

Reply:
Done!